src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php

```diff
@@ -0,0 +1,17 @@
+<h1>Build #1615360383</h1>
+<h2>Date: 2021-03-10</h2>
+<div class="changelog">
+    - #692: Fix active icon list display
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'de')
+  ->setFields([
+      'callback_fnc' => 'gcf_GetAciveIcon',
```

Comment:
Shouldn't that be `getActiveIcon`?

Reply:
Nope. The prefix has matching reasons. gcf_ = global callback function (core), ccf_ = userspace

Reply:
Thinking about this... this file based function repository is kind of deprecated. It is possible to add callback functions to Tdb record classes. If there is a getActiveIcon method this is the one, it should reference.

Reply:
I was not addressing the prefix, but the missing `t` in `Acive` :)

Reply:
Oh... never mind... :-)

Reply:
getAciveIcon is how it is named...

Reply:
Oh my...